.github/workflows/docs-deploy.yml

```diff
@@ -0,0 +1,75 @@
+name: "Deploy Docs"
+on:
+  push:
+    branches: [master]
+
+jobs:
+  # https://github.com/marketplace/actions/deploy-to-github-pages
+  build-docs-deploy:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2
+        # If you're using actions/checkout@v2 you must set persist-credentials to false in most cases for the deployment to work correctly.
+        with:
+          persist-credentials: false
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - id: 'auth'
+        name: 'Authenticate to Google Cloud'
+        uses: 'google-github-actions/auth@v0'
+        with:
+          credentials_json: ${{ secrets.GCS_SA_KEY }}
```

Comment:
do we have this configured?

Reply:
Yes, this was added by Lexie. But we can try this only once merged.